nengo/networks/workingmemory.py

```diff
@@ -5,15 +5,13 @@
 
 
 def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+                     difference_gain=1.0, reset_gain=3, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    mem_config = nengo.Config(nengo.Connection)
+    mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
```

Comment:
If the config parameter is removed (as I completely agree with), this synapse parameters should be exposed.  I'd recommend an argument like ```recurrent_synapse``` and have the default be 0.1.  Hmm, although it looks like the network also uses that same value for the connection from the difference output to the memory input.  Should those be the same value?  Should that be a separate parameter?  If they should be the same value, then is there a better parameter name than ```recurrent_synapse```?